Assets/Scripts/SceneLoader.cs

```diff
@@ -0,0 +1,30 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+using UnityEngine.SceneManagement;
+
+public class SceneLoader : MonoBehaviour
+{
+    public int sceneNumber;
+
+    // Start is called before the first frame update
+    void Start()
+    {
+        
+    }
+
+    // Update is called once per frame
+    void Update()
+    {
+        
+    }
+
+    /// <summary>
+    /// Loads the scene number based on the integer value
+    /// that is given to this script.
+    /// </summary>
+    public void ChangeScene()
+    {
+        SceneManager.LoadScene(sceneNumber);
```

Comment:
If you make it so SceneNumber is a param on the function then only one object actually needs to have this script on it and the buttons can call this from there